src/banner.js

```diff
@@ -43,6 +43,7 @@ export function insertCookieBanner(onAccept) {
     removeFocusCookieConfirmation();
   });
   document.getElementById('nhsuk-cookie-banner__link').addEventListener('click', () => {
+    onAccept();
```

Comment:
This means that banner won't be reopened again, despite the user not taking an action to close it. We need to somehow make it so the banner only doesn't appear on one specific page, either by another cookie -_- or something in the cookie toggles component to disable the banner on that page?

Reply:
@bradmorton1 you're right.
Maybe we don't make a change here at all, and just use  the "nobanner" mode on the cookie toggles page.
`<script src="./cookie-consent.js" data-nobanner type="text/javascript"></script>`

What do you think?